src/test/ui/consts/const-eval/infinite_loop.stderr

```diff
@@ -14,32 +14,19 @@ LL | |         }
    = help: add `#![feature(const_if_match)]` to the crate attributes to enable
 
 error[E0658]: `if` is not allowed in a `const`
-  --> $DIR/infinite_loop.rs:9:17
+  --> $DIR/infinite_loop.rs:8:17
    |
 LL |             n = if n % 2 == 0 { n/2 } else { 3*n + 1 };
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #49146 <https://github.com/rust-lang/rust/issues/49146> for more information
    = help: add `#![feature(const_if_match)]` to the crate attributes to enable
 
-warning: Constant evaluating a complex constant, this might take some time
```

Comment:
This may be useful to keep around.

Reply:
You mean at like 1% or 10% of the step limit? Not sure how to choose a good number for that, but anything above 10% of the step limit seems useless since you're hitting the step limit soon, and anything below 30s will likely be annoying more often than not. It's very hard to silence this warning (it may be getting emitted in a different crate from the one currently being evaluated).

Reply:
Anecdotally, the default step limit only takes a few seconds to trigger, so I don't think preserving the warning is helpful.